onyo/lib/commands.py

```diff
@@ -65,19 +82,40 @@ def fsck(repo: OnyoRepo, tests: Optional[list[str]] = None) -> None:
 
     # run the selected tests
     for key in tests:
-        # TODO: these should be INFO
-        ui.log_debug(f"'{key}' starting")
+        ui.log(f"'{key}' starting")
 
         if not all_tests[key]():
-            # Note: What's that debug message adding? Alone it lacks the identifying path and in combination with
-            #       the exception it's redundant.
+            # Note: What's that debug message adding? Alone it lacks the
+            #       identifying path and in combination with the exception
+            #       it's redundant.
             ui.log_debug(f"'{key}' failed")
             raise OnyoInvalidRepoError(f"'{repo.git.root}' failed fsck test '{key}'")
 
-        ui.log_debug(f"'{key}' succeeded")
+        ui.log(f"'{key}' succeeded")
 
 
-def onyo_cat(repo: OnyoRepo, paths: Iterable[Path]) -> None:
+def onyo_cat(repo: OnyoRepo,
+             paths: Iterable[Path]) -> None:
+    """Run the command `onyo cat` to display the contents of assets.
```

Comment:
That's a strange phrasing. This function is not "running the command" it *is* the command.
The description should state what it does.

```suggestion
    """Print the contents of assets.
```